audio/aacgain/Portfile

```diff
@@ -33,6 +33,7 @@ configure.args      --without-x
 
 platform darwin {
     configure.cflags-append -DHAS_LRINTF
+    configure.cxxflags-append -std=c++03
```

Comment:
This addition should be outside of this conditional. The code is written to C++03 regardless what platform we try to compile it on.

Also set `compiler.cxx_standard 2003` for good measure.